Enable clone finished game and auto prune

vincentneko opened the pull request:
- Finished games will now appear in the clonable game list
- Auto pruning of unfinished games. When a game is finished, a query will be launched to purge other unfinished games that are older than 10 days.
@bafolts this should reduce the memory footprint and maybe avoid to manually purge the table. 10 days is maybe too high, we can change this if needed.

bafolts commented on src/database/PostgreSQL.ts:
```diff
@@ -130,8 +130,14 @@ export class PostgreSQL implements IDatabase {
+            });           
         });
+        // Purge unfinished games older than 10 days
+        this.client.query("DELETE FROM games WHERE created_time > now() - interval '10 days' and status = 'running'", function(err: { message: any; }) {
```
Should this be `created_time < now() - interval '10 days'`? I assume this would delete games _newer_ than 10 days. I will have to double check this.

vincentneko replied:
Yes you are right ! Will update asap

vincentneko commented:
> Don't merge this until the postgres query is fixed. As written this will delete games newer than 10 days old.

Updated.
I've also added an index on created_time column (@bafolts I think you already created this one)

alrusdi commented on src/database/PostgreSQL.ts:
```diff
@@ -30,7 +30,13 @@ export class PostgreSQL implements IDatabase {
+            if (err) {
+                throw err;
+            }
+            console.log("Connected to PostgreSQL");
```
Is it message really needed?

vincentneko replied:
> Is it message really needed?

Nope, just removed it

bafolts commented on src/database/PostgreSQL.ts:
```diff
@@ -130,8 +133,14 @@ export class PostgreSQL implements IDatabase {
+            });           
         });
+        // Purge unfinished games older than 10 days
+        this.client.query("DELETE FROM games WHERE created_time < now() - interval '10 days' and status = 'running'", function(err: { message: any; }) {
```
I will sanity check this query. I wouldn't assume these queries are running in series. The call to `this.client.query` is asynchronous and these queries may end up running after the function returns. I don't think this is a problem for our use case here but if we continue to add queries it could lead to difficult to track down bugs.
